builtin/providers/azurerm/data_source_arm_resource_group.go

```diff
@@ -0,0 +1,49 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceArmResourceGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmResourceGroupRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"location": locationForDataSourceSchema(),
+			"tags":     tagsForDataSourceSchema(),
+		},
+	}
+}
+
+func dataSourceArmResourceGroupRead(d *schema.ResourceData, meta interface{}) error {
+	armClient := meta.(*ArmClient)
+
+	resourceGroupName := d.Get("name").(string)
+	location, getLocationOk := d.GetOk("location")
+	resourceId := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", armClient.subscriptionId, resourceGroupName)
```

Comment:
Can we create a helper func for this? It can look something like:

```
func buildResourceId(meta interface{}, resourceType string, resourceName string) string {

}
```